spec/services/create_form_service_spec.rb

```diff
@@ -25,5 +25,63 @@
 
       expect(GroupForm.last).to have_attributes(form_id: 1, group_id: 1000)
     end
+
+    # rubocop: disable RSpec/ContextWording
```

Comment:
Is there the need to disable this rule? I think "when" makes more sense than "if" for the outer context, and the other contexts could be worded to include a "when" and would probably be easier to read in the context of reading this file than without.

Reply:
Hmm, I struggled with this... an example would help... what would you suggest rewriting this to?

> if a form with that name was already created in that group at the same time

Reply:
"when a form with the same name was already created in the same group"
> "when both forms are created at the same time"
> "when the previous form was created less then one second ago"
> "when the previous form was created more than one second ago"
> "when the previous form was created by a different user"

It's a bit more verbose. But, personally I'd prefer if we either disabled this rule globally if we don't think it's helpful or followed it so we have consistency.

Reply:
Makes sense to me. I like the rule in general, I've followed your suggestion to make it more consistent in these specs.